modules/seqexec/server/src/main/scala/seqexec/server/ODBProxy.scala

```diff
@@ -5,54 +5,55 @@ package seqexec.server
 
 import cats.data.EitherT
 import cats.effect.IO
+import cats.implicits._
 import edu.gemini.pot.sp.SPObservationID
```

Comment:
I think this is the only reference left to ocs2 `SPObservationID` as we need to talk to the old odb. Otherwise we use `Observation.Id` everywhere else